### PR TITLE
July Update

### PR DESCRIPTION
**Pull request by erogluorhan**

### CODE REVIEW

File: blog/_posts/2021-08-18-July-2021-update.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Dear GeoCAT Community,
+
+Our team would like to share July, 2021 update 
+about GeoCAT releases and community activities below:  
+
+### Software Releases
+
+#### - GeoCAT-comp v2021.07.0 and v2021.07.1 Released!
+With these releases, new functions such as `calender_average` and `climatology_average`
```
Comment (michaelavs) [inline blog/_posts/2021-08-18-July-2021-update.md]:
```suggestion
With these releases, new functions such as `calendar_average` and `climatology_average`

```

File: blog/_posts/2021-08-18-July-2021-update.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+for detailed information about our development stack. 
+
+### Community Events
+
+In addition to our regular development activities, the GeoCAT team engaged
+with the community through a few virtual events in July.
+
+#### - SIParCS 2021
+NCAR's SIParCS 2021 internship program ended on July 30th. The
+GeoCAT team thanks to our three interns (Erin Lincoln, Jiaqi Li, Rose Craker)
```
Comment (michaelavs) [inline blog/_posts/2021-08-18-July-2021-update.md]:
```suggestion
GeoCAT team thanks to our three interns (Erin Lincoln, Jiaqi Li, Heather Craker) 

```

Comment (erogluorhan) [reply blog/_posts/2021-08-18-July-2021-update.md]: Done

File: blog/_posts/2021-08-18-July-2021-update.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Our team would like to share July, 2021 update 
+about GeoCAT releases and community activities below:  
+
+### Software Releases
+
+#### - GeoCAT-comp v2021.07.0 and v2021.07.1 Released!
+With these releases, new functions such as `calendar_average` and `climatology_average` 
+under the `climatologies` module have been made available; improvements to Dask compatibility 
+on several functions such as `dewtemp`, `heat_index`, and `relhum` function family have been
+performed; a few bug fixes have been addressed; some codes have been refactored for improved
```
Comment (anissa111) [inline blog/_posts/2021-08-18-July-2021-update.md]:
```suggestion
under the `climatologies` module have been made available; improvements to Dask compatibility 
on several functions have been
made ; a few bug fixes have been addressed; some codes have been refactored for improved

```

**Review by anissa111 — APPROVED** [review]
One sentence I noted below reads a little funny, but this looks good!

File: blog/_posts/2021-08-18-July-2021-update.md (hunk 3 above)
Comment (erogluorhan) [reply blog/_posts/2021-08-18-July-2021-update.md]: Thanks, done!

**Pull request merged**